jupyter_rfb/_colab.py

```diff
@@ -0,0 +1,31 @@
+try:
+    # determine if running in colab
+    from google.colab import output
+    from google.colab.output._widgets import _installed_url
+except Exception:
+    IN_COLAB = False
+    COLAB_URL = None
+else:
+    IN_COLAB = True
+    # useful to enable widget manager here so the user doesn't have to manually
+    output.enable_custom_widget_manager()
+    # the metadata that colab needs
+    COLAB_URL = _installed_url
```

Comment:
I don't think this will work, just like how `bar` doesn't get updated when `foo` is reassigned below:

```python
foo = "bar"
bar = foo
foo = "baz"
print(bar)
# "bar"
```

Larger example:

```python
# bar.py
baz = "Ooops, still bar"

def switch_baz():
    global baz
    baz = "It worked"
```

```python
# foo.py
from bar import baz, switch_baz

print(baz)
switch_baz()
print(baz)
```

```sh
python3 foo.py
# Ooops, still bar
# Ooops, still bar
```

I'd check for `IN_COLAB` and then import the `COLAB_URL` in the `get_colab_metadata`.

Reply:
Ah, so `_widgets._installed_url` is set after creating the widget?